nginx/metrics-nginx.yaml

```diff
@@ -0,0 +1,133 @@
+---
+# Description
+#   Provide NGINX metrics through status module
```

Comment:
Same comment as above – let's make a more verbose reference to the underlying NGINX dependency.